sympy/printing/tests/test_julia.py

```diff
@@ -0,0 +1,365 @@
+from sympy.core import (S, pi, oo, symbols, Function, Rational, Integer,
+                        Tuple, Symbol)
+from sympy.core import EulerGamma, GoldenRatio, Catalan, Lambda
+from sympy.functions import Piecewise, sqrt, ceiling, exp, sin, cos
+from sympy.utilities.pytest import raises
+from sympy.utilities.lambdify import implemented_function
+from sympy.matrices import (eye, Matrix, MatrixSymbol, Identity,
+                            HadamardProduct, SparseMatrix)
+from sympy.functions.special.bessel import (jn, yn, besselj, bessely, besseli,
+                                            besselk, hankel1, hankel2, airyai,
+                                            airybi, airyaiprime, airybiprime)
+from sympy.utilities.pytest import XFAIL
+from sympy.core.compatibility import range
+
+from sympy import julia_code
+from sympy import julia_code as mcode
```

Comment:
This is confusing. I'm guessing this was done because you copied this from the octave printer. I'd rather just test against "julia_code".